conf/maven/pom.xml

```diff
@@ -239,7 +239,7 @@
   <properties>
     <swagger-core-version>2.2.6</swagger-core-version>
     <jersey-version>2.37</jersey-version>
-    <jackson-version>2.14.0</jackson-version>
```

Comment:
These changes are not relative to the title of the commit and the PR
Please add a specific commit